_rules/line-height-not-important-78fd32.md

```diff
@@ -28,7 +28,7 @@ This rule applies to any HTML element that is [visible][] and for which the `sty
 For each test target, at least one of the following is true:
 
 - **not important**: the [computed][] value of its [line-height][] property is not [important][]; or
-- **large enough**: the [computed][] value of its [line-height][] property is not `normal`, and is at least `1.5` or 1.5 times the [computed][] value of its [font-size][] property; or
+- **large enough**: the [computed][] value of its [line-height][] property is not `normal`, and is at least 1.5 times the [computed][] value of its [font-size][] property; or
```

Comment:
It's actually not duplicated…
Computed `line-height` can be a unitless number, with the meaning "X times the `font-size`", or a length (with unit).

```
<p style="font-size: 16px; line-height: 10 !important">Foo</p>
```
has a **computed** `line-height` of `10`, which is not 1.5 times the `font-size` (`16px`) (notably because a unitless number is not comparable to a unit), yet it is nonetheless OK.

Maybe saying that the **used** value is 1.5 times the `font-size` is good (https://drafts.csswg.org/css-inline/#line-height-property seems down right now, so I can't really check that). But I remember we spent a lot of time chiselling the conditions in that rule, so I'd be very careful before making such a change.

Reply:
I see your point, but I think that the phrasing was confusing. I will checkout the link when it is back up, but in the meantime, could we say something like "at least the unitless constant `1.5` or 1.5 times...", something that disambiguates the first 1.5 from the second.

Reply:
Yes, I'm not saying it shouldn't be changed 😄
Improving the formulation is definitely possible, but we do need to keep something to handle the unitless numbers.